src/gui/sdlmain.cpp

```diff
@@ -2145,18 +2151,33 @@ static void GUI_StartUp(Section *sec)
 	std::string focus = p->GetSection()->Get_string("active");
 	std::string notfocus = p->GetSection()->Get_string("inactive");
 
-	if      (focus == "lowest")  { sdl.priority.focus = PRIORITY_LEVEL_LOWEST;  }
-	else if (focus == "lower")   { sdl.priority.focus = PRIORITY_LEVEL_LOWER;   }
-	else if (focus == "normal")  { sdl.priority.focus = PRIORITY_LEVEL_NORMAL;  }
-	else if (focus == "higher")  { sdl.priority.focus = PRIORITY_LEVEL_HIGHER;  }
-	else if (focus == "highest") { sdl.priority.focus = PRIORITY_LEVEL_HIGHEST; }
-
-	if      (notfocus == "lowest")  { sdl.priority.nofocus=PRIORITY_LEVEL_LOWEST;  }
-	else if (notfocus == "lower")   { sdl.priority.nofocus=PRIORITY_LEVEL_LOWER;   }
-	else if (notfocus == "normal")  { sdl.priority.nofocus=PRIORITY_LEVEL_NORMAL;  }
-	else if (notfocus == "higher")  { sdl.priority.nofocus=PRIORITY_LEVEL_HIGHER;  }
-	else if (notfocus == "highest") { sdl.priority.nofocus=PRIORITY_LEVEL_HIGHEST; }
-	else if (notfocus == "pause")   {
+	if (focus == "default") {
+		sdl.priority.focus = PRIORITY_LEVEL_DEFAULT;
```

Comment:
Looking at the surrounding brackets, it looks like this if-else chain is stand-alone, so I think the brackets can be stripped from it (minor.. but a bit less clutter might make the block easier on the eyes). 

This is the one area where we make the formatter trade off visual appeal (allowing branch points and code on the same line) for hard consistently: readers can be guaranteed code always comes after the branch point, and don't have to hunt around for code that can be hiding on the same line of huge line of conditionals.

Reply:
ok, so, just so I'm clear:

```
	if (focus == "auto")
		sdl.priority.focus = PRIORITY_LEVEL_AUTO;
	else if (focus == "lowest")
		sdl.priority.focus = PRIORITY_LEVEL_LOWEST;
	else if (focus == "lower")
		sdl.priority.focus = PRIORITY_LEVEL_LOWER;
	else if (focus == "normal")
		sdl.priority.focus = PRIORITY_LEVEL_NORMAL;
	else if (focus == "higher")
		sdl.priority.focus = PRIORITY_LEVEL_HIGHER;
	else if (focus == "highest")
		sdl.priority.focus = PRIORITY_LEVEL_HIGHEST;

	if (notfocus == "auto")
		sdl.priority.nofocus = PRIORITY_LEVEL_AUTO;
	else if (notfocus == "lowest")
		sdl.priority.nofocus = PRIORITY_LEVEL_LOWEST;
	else if (notfocus == "lower")
		sdl.priority.nofocus = PRIORITY_LEVEL_LOWER;
	else if (notfocus == "normal")
		sdl.priority.nofocus = PRIORITY_LEVEL_NORMAL;
	else if (notfocus == "higher")
		sdl.priority.nofocus = PRIORITY_LEVEL_HIGHER;
	else if (notfocus == "highest")
		sdl.priority.nofocus = PRIORITY_LEVEL_HIGHEST;
	else if (notfocus == "pause")
		/* we only check for pause here, because it makes no sense
		 * for DOSBox to be paused while it has focus
		 */
		sdl.priority.nofocus=PRIORITY_LEVEL_PAUSE;
```

?

Reply:
Yes, looks good!